go/ql/lib/change-notes/2024-08-12-add-file-models.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Local source models have been added for the file APIs in the "os" package in the Go standard library.
```

Comment:
I also recommend adding some information about threat models, as this is a new feature. You can copy the last sentence or two from [here](https://github.com/github/codeql/pull/16760/files#diff-e4d8dae8f74704a02f34a0b9ddd094dd24f768eee4a75cffad3a150d352eb140), maybe.